.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,54 @@
+name: publish-release
+
+on: workflow_dispatch
```

Comment:
This allows us to manually trigger the "publish-release" workflow. Now,

- "publish-pre-release" will run on every merge to main, creating x.y.z-rc.N, x.y.z-rc.M, etc.
- "publish-release" will be manually triggered when we are ready to release.